CSS3.sublime-syntax

```diff
@@ -10824,15 +10824,14 @@ contexts:
   selector:
     # This match applies a single meta.selector.css scope to a comma-separated
     # list of selectors, so that they appear as one entry in the Symbol List.
-    - match: '(?=[\w.:#\[*-&]|@nest|@layer)'
+    - match: '(?=[\w.:#\[*-&]|@layer)'
```

Comment:
I honestly have no idea what this line does and why it includes an `@layer` rule - perhaps there are more changes to be made here. Do let me know 🙂

Reply:
not sure either, i can take a peek, it might be me that put it there